application/forms/Command/Instance/ToggleInstanceFeaturesForm.php

```diff
@@ -37,6 +45,13 @@ public function __construct($featureStatus)
         ];
 
         $this->getAttributes()->add('class', 'instance-features');
+
+        $this->on(self::ON_SUCCESS, function () {
+            $status = $this->submittedFeature->getEnabled() ? 'enabled' : 'disabled';
+            $label = $this->features[$this->submittedFeature->getFeature()];
+
+            Notification::success(sprintf(t('%s %s successfully'), $label, $status));
```

Comment:
`Enabled active checks successfully`, `Disabled active checks successfully`. One for each feature and state, both must be a literal in the message otherwise it cannot be translated.

Also, there may be multiple features toggled at the same time. Currently only for the last command the notification is shown. It's not possible to toggle multiple features at the same time via the UI, but the implementation already supports that in case we have a API some time where this will in fact be possible. So please also consider this when registering notifications.